miner/active_set_generator_test.go

```diff
@@ -58,8 +58,8 @@ type test struct {
 	expectErr string
 }
 
-func unixPtr(sec, nsec int64) *time.Time {
-	t := time.Unix(sec, nsec)
+func unixPtr(sec int64) *time.Time {
+	t := time.Unix(sec, 0)
 	return &t
 }
```

Comment:
This function isn't needed in my opinion. `epochStart` can just be a value instead of a pointer and instead of checking it for `nil` one can call `epochStart.IsZero()` on it to see if it was set or not.

Wdyt?